Makefile

```diff
@@ -12,6 +12,8 @@
 # # See the License for the specific language governing permissions and
 # # limitations under the License.
 
+include ./vendor/github.com/openshift/build-machinery-go/make/targets/openshift/deps.mk
```

Comment:
I think it would be useful to add a comment here to note that this adds the `verify-deps` target.